packages/cysync-core/src/dialogs/Send/index.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  DialogBox,
+  DialogBoxBody,
+  Container,
+  WalletDialogMainContainer,
+  MilestoneAside,
+  CloseButton,
+  BlurOverlay,
+} from '@cypherock/cysync-ui';
+import React, { FC } from 'react';
+
+import {
+  closeDialog,
+  selectLanguage,
+  useAppDispatch,
+  useAppSelector,
+} from '~/store';
+
+import { SendDialogProvider, useSendDialog } from './context';
+
+export const SendFlow: FC = () => {
+  const { tabs, currentTab, currentDialog } = useSendDialog();
+  const dispatch = useAppDispatch();
+  const lang = useAppSelector(selectLanguage);
+
+  return (
+    <BlurOverlay>
+      <DialogBox direction="row" gap={0} width="full">
+        <>
+          <MilestoneAside
+            milestones={tabs.map(t => t.name)}
+            heading={lang.strings.send.aside.tabs.heading}
+            activeTab={currentTab}
+          />
+          <WalletDialogMainContainer>
+            <Container width="full" p={2} justify="flex-end">
+              <CloseButton
+                onClick={() => dispatch(closeDialog('sendDialog'))}
+              />
+            </Container>
```

Comment:
Refer `dialogs/GuidedFlow/index.tsx` for adding onClose

Reply:
done - with CloseConfirmation dialog created